packages/react-search-ui-views/src/PagingInfo.js

```diff
@@ -1,17 +1,27 @@
 import PropTypes from "prop-types";
 import React from "react";
 
-import { appendClassName } from "./view-helpers";
+import { appendClassName, ScreenReaderStatus } from "./view-helpers";
```

Comment:
One potential downside to adding this to the `<PagingInfo />` component is that if users roll their own view, they're inadvertently making their app less accessible - so it's possible that adding this to the `react-search-ui` container may be a bit more robust. Definitely open to thoughts on this!

Reply:
In this project it's often a bit fuzzy to me what should go into a container and what should go into a view. 

The guidance we give users for customizing views is to use the default view as a base and customize from there. That could be a bit of a foot gun if they forget to call "announceToScreenReader", as you point out.

The goal of having swappable views, is to give users a very easy way to use custom markup for a component. The less they have to reproduce in the view, the better. To that end, it would seem more ideal to do this in a container. We could either do it completely transparently, or we could roll all of the logic into a single prop that we pass down.

```jsx
function PagingInfo({ className, end, searchTerm, start, totalResults, screenReaderAnnouncement }) {
  end = Math.min(end, totalResults);
  return (
    <>
      <div className={appendClassName("sui-paging-info", className)}>
        Showing{" "}
        <strong>
          {start} - {end}
        </strong>{" "}
        out of <strong>{totalResults}</strong> for: <em>{searchTerm}</em>
      </div>
      {screenReaderAnnouncement(someCustomMessageFunction)}
    </>
  );
}
```

It could be something like, if you don't call it at all, the default announcement will be used. if you call it with a custom function, `screenReaderAnnouncement()`, then that overrides the default message with a custom message.

Reply:
I also envisioned a future state where users could use `react-search-ui` without including `react-search-ui-views` at all, so they could have a completely custom view layer. Having this be part of the container would make that a challenge 🤔.